task/sanity-label-check/0.1/README.md

```diff
@@ -0,0 +1,25 @@
+# sanity-label-check task
+
+## Description:
+The sanity-label-check task utilizes conftest to verify label data. Conftest is an open-source tool that provides a way
+to test and enforce compliance with policies written in a high-level declarative language called Rego.
+
+Conftest can be used as part of a CI/CD pipeline to ensure that configuration files are compliant with a set of policies and best practices before they are deployed to production. It provides a way to catch configuration issues early in the development process and reduce the risk of misconfigurations or other issues in production.
+
+## Params:
+
+| name             | description                                         |
+|------------------|-----------------------------------------------------|
+| POLICY_DIR       | Path to the directory containing conftest policies. |
+| POLICY_NAMESPACE | Namespace for the conftest policy.                  |
+| DOCKER_AUTH      | Secret with config.json for container auth.         |
+
+## Results:
+
+| name                  | description                                |
+|-----------------------|--------------------------------------------|
+| HACBS_TEST_OUTPUT     | Test output.                               |
+
```

Comment:
I think it might be good to also add a link to the source repository, something like this:
```
## Source repository for image:
https://github.com/redhat-appstudio/hacbs-test
```